plugin/core/sessions.py

```diff
@@ -833,10 +836,17 @@ def on_transport_close(self, exit_code: int, exception: Optional[Exception]) ->
         self.state = ClientStates.STOPPING
         super().on_transport_close(exit_code, exception)
         self._response_handlers.clear()
+        if self._init_error:
+            init_code, init_exception = self._init_error
+            exit_code = init_code
+            exception = init_exception
```

Comment:
Where are the variables `exit_code` and `exception` used?

Reply:
Below, in the call to `mgr.on_post_exit_async`